package/Dockerfile.dapper-base

```
@@ -41,3 +41,7 @@ RUN dnf -y distrosync && \
     curl -Lo /usr/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-${ARCH}" && chmod a+x /usr/bin/kind && \
     GOFLAGS="" go get -v github.com/onsi/ginkgo/ginkgo && \
     GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports
+
+# Copy CI deployment scripts into image to share with all submariner-io/* projects
+WORKDIR /opt/kind-e2e
+COPY scripts/kind-e2e/ /opt/kind-e2e/
```

```suggestion
COPY scripts/kind-e2e/ .
```

This should work since you just set the workdir, no need to specify it again :)

---

Ah yes, that makes sense, done and thanks!